CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 
+# 0.20.0
+
+* The command timeout duration of a Rivet operation will be set to the value specified in the `CommandTimeout` property

```

Comment:
Change version to `0.19.1` and rephrase: "Fixed: Rivet doesn't use the CommandTimeout property in rivet.json configuration file."